### PR TITLE
Add blog Joep from America

### PR DESCRIPTION
**Pull request by sdegeus**

### CODE REVIEW

**Comment by DavidFritsch** [comment]
- [ ] Add Images
- [ ] Add hyperlink

File: source/blog/nl/2017-11-16-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+lang: nl
+featured: true
+image: images/blog/social/masie-learning-2017.jpg
+featured_image: images/blog/featured/20171120-masie-learning-2017.jpg
+---
+
+Het is inmiddels al weer twee weken geleden dat ik in Orlando het [Learning2017 congres](https://www.learning2017.com){:target="_blank"}{:rel="noopener"}{:title="Learning 2017"} van Eliott Masie bezocht. Toen beschreef ik [hier](/blog/masie-over-masie-learning-2017/) al het begin van het congres. Nog steeds duizelt het me van alle indrukken en inspiratie die dit congres opgeleverd heeft. Voordat ik in een serie van blogs dieper op specifieke zaken in ga, wil ik graag mijn belangrijkste take aways van het congres met jullie delen.
+
+![Julian Stodd & Eliott Maise](/images/blog/social/masie-learning-2017.jpg)
+![Milda, Stéphanie & Marc Buckley](/images/blog/learning-2017-escape-room.jpg)
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-11-16-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown]: Update alt text bitte

File: source/blog/nl/2017-11-16-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Neem bijvoorbeeld een willekeurige leerinterventie. Op een afdeling gaat een bepaald proces relatief vaak mis. Er wordt besloten dat de afdeling moet worden bijgeschoold in het proces in kwestie. In het LMS wordt vervolgens vastgelegd wie allemaal op de training zijn geweest. Dit wordt gerapporteerd en daarmee is aangetoond dat er actie is ondernomen. Hiermee zeggen we echter nog niets over de kwaliteit van de interventie. Ook tonen we niet aan of het probleem nu is opgelost. Worden er na de interventie ook daadwerkelijk minder fouten gemaakt in het proces? We hopen natuurlijk van wel. Dit zouden we o.a. kunnen aan te tonen door fouten voor en na de interventie te vergelijken, de kwaliteit van het werk van de medewerkers die uitblonken correleren met hun scores op de leerinterventie. Door systemen data met elkaar uit te laten wisselen is dit allemaal niet eens heel lastig om te regelen. Dit soort data is veel relevanter voor je organisatie en helpt mee om het belang van een academie binnen een grote organisatie kwantificeerbaar te maken. Wie weet leidt dit op zijn beurt wel weer tot een groter budget en meer mogelijkheden wanneer je je eigen meerwaarde aantoont...
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-11-16-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown]: Extra enter please

File: source/blog/nl/2017-11-23-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+date: 2017-11-23 8:00 CEST
+tags: conference, innovatie
+author: Joep
+lang: nl
+featured: true
+image: images/blog/social/masie-stodd-learning-2017.jpg
+featured_image: images/blog/featured/20171120-masie-learning-2017.jpg
+---
+
+Het is inmiddels al weer twee weken geleden dat ik in [Orlando het Learning2017 congres van Eliott Masie bezocht](/masie-over-masie-learning-2017/). Nog steeds duizelt het me van alle indrukken en inspiratie die dit congres opgeleverd heeft. Voordat ik in een serie van blogs dieper op specifieke zaken in ga, wil ik graag mijn belangrijkste take aways van het congres met jullie delen.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-11-23-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown]: alweer = al weer

Comment (RonaldDijkstra) [inline source/blog/nl/2017-11-23-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown]: I prefer Learning 2017 congres van Eliott Masie in Orlando.

Comment (RonaldDijkstra) [inline source/blog/nl/2017-11-23-masie-de-drie-belangrijkste-takeaways-uit-orlando.html.markdown]: Link doesn't work, that should be `/blog/masie-over-masie-learning-2017/`, relative to root (defacto.nl) -> see html-proofer test

**Pull request merged**